tests/conftest.py

```diff
@@ -23,3 +21,20 @@ def check(output: str) -> bool:
     else:
         with open(compare_file, "r") as infile:
             assert yaml_output == infile.read()
+
+
+@pytest.fixture
+def gitlab_ci_environment_variables(monkeypatch):
+    """
+    Fixture to patch GitLab CI predefined environment variables.
+
+    You can extend this fixture with all GitLab CI environment variables which can be used form gcip.PredefinedVariables.
```

Comment:
```suggestion
    You can extend this fixture with all GitLab CI environment variables that will be used by gcip.PredefinedVariables.
```

Reply:
I did change that and except your suggestgion. Thank you.